src/pages/artikler/index.astro

```diff
@@ -0,0 +1,60 @@
+---
+import Layout from '@layouts/default.astro';
+import { getCollection } from 'astro:content';
+import PostCard from '@components/post-card.astro';
+import Brief from '@components/brief.astro';
+
+const posts = await getCollection('post');
+
+const compareNum = (a: any, b: any) => {
+  if (a.data.date.valueOf() > b.data.date.valueOf()) {
+    return -1;
+  } else if (a.data.date.valueOf() < b.data.date.valueOf()) {
+    return 1;
+  }
+  return 0;
+}
```

Comment:
As far as I can tell data.date is a Date object, which means we should be able to compare them directly like this:
```suggestion
const compareNum = (a: any, b: any) => {
  if (a.data.date > b.data.date) {
    return -1;
  } else if (a.data.date < b.data.date) {
    return 1;
  }
  return 0;
}
```